.github/workflows/publish-docs.yml

```diff
@@ -11,40 +11,123 @@ on:
         description: "The commit SHA, tag, or branch to publish. Uses the default branch if not specified."
         default: ""
         type: string
+  workflow_call:
+    inputs:
+      plan:
+        required: true
+        type: string
 
 jobs:
   mkdocs:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
```

Comment:
Why do we need these permissions if we're writing to another repository with a dedicated secret token?

Reply:
Sorry, we don't, this is copied over from Prefect.